quickstart/go/Dockerfile

```diff
@@ -0,0 +1,24 @@
+# Use the offical Golang image to create a build artifact.
+# This is based on Debian and sets the GOPATH to /go.
+# https://hub.docker.com/_/golang
+FROM golang:1.12 as builder
+
+# Copy local code to the container image.
+WORKDIR /go/src/github.com/knative/docs/helloworld
+COPY . .
+
+# Build the command inside the container.
+# (You may fetch or manage dependencies here,
+# either manually or with a tool like "godep".)
```

Comment:
I'd remove the reference to downloading dependencies and `godep`. If we do reference dependencies, we should show copying just the `go.mod` and `go.sum` files, running `go mod download`, copying everything else, then doing `go build`.

cc @grayside